README.md

```
@@ -1,10 +1,34 @@
+<a href="https://github.com/wtasg/meetonline/blob/main/LICENSE">
+  <img src="https://img.shields.io/github/license/wtasg/meetonline?style=for-the-badge&color=green" />
+</a>
+
+</p>
+
 An app to let you build and find online communities.
 
 ## Reporting bugs
 
-- Read up, copy-paste from [Bug Report template](./docs/bug-report-template.md)
+-   Read up, copy-paste from [Bug Report template](./docs/bug-report-template.md)
```

markdownlint-cli2 (0.18.1)

Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)